src/content/docs/jellyfin-integration/installing-shokofin.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Installing Shokofin
+sidebar:
+  order: 1
+Description: Information on how to install and configure Shokofin (Jellyfin)
+---
+
+import EasyTable from "../../../components/EasyTable/EasyTable";
+
+**Shokofin** is an anime metadata provider plugin for **Jellyfin** that works in tandem with **Shoko Server** to provide
+metadata tailored to your personal anime collection. For information on what Shoko Server is and the benefits that
+can come from using it to identify and manage your collection, visit our [Official Website](https://shokoanime.com/).
+
+## Requirements
+
+Shokofin is not meant to be used stand-alone and requires Shoko Server to be installed, configured, and running before
+it can pull in metadata specific to your library. If you haven't done so, [Download Shoko Server](https://shokoanime.com/downloads/)
+from our main website and use the [Installing Shoko Server](../../../getting-started/installing-shoko-server) guide to
+properly install and configure Shoko Server.
+
+Shokofin requires Shoko Server to already be setup and in use, and that the directories/folders you intend to
+use in Jellyfin are fully indexed (and optionally managed) by Shoko Server. Without doing so beforehand, Shokofin will be
+unable to identify any of your anime series.
+
+## Install
+
+Below lists the corresponding versions of Jellyfin and Shoko Server that are compatible with Shokofin. Be sure to install
+the correct version for your setup to guarantee functionality.
+
+<EasyTable
+    header={["Shokofin", "Jellyfin", "Shoko Server"]}
+    data={[
+      ["`0.x.x`",     "`10.7`",   "`4.0.0`&mdash;`4.1.2`"],
+      ["`1.x.x`",     "`10.7`",   "`4.1.0`&mdash;`4.1.2`"],
+      ["`2.x.x`",     "`10.8`",   "`4.1.2`"              ],
+      ["`3.x.x`",     "`10.8`",   "`4.2.0`"              ],
+      ["`unstable`",  "`10.8`",   "`daily`"              ],
```

Comment:
and i just made the "unstable" version compatible with 4.2.2 of the stable server again. I'll update the read me on GH tomorrow, but now you know it will be updated.

Reply:
(this is mostly because JF 10.9 is around the corner and the next version of Shoko Server is not. so it meeds to be compatible)